src/coordsim/flow_generators/default_generator.py

```diff
@@ -0,0 +1,36 @@
+import logging
+import numpy as np
+import random
+from typing import Tuple
+from coordsim.network.flow import Flow
+from coordsim.flow_generators import BaseFlowGenerator
+log = logging.getLogger(__name__)
+
+
+class DefaultFlowGenerator(BaseFlowGenerator):
```

Comment:
This should be independent of the algorithm we use for SPR, right? But still doesn't hurt to have it modular I guess. Or do you have any specific other child classes in mind?

Reply:
I thought maybe another algorithm might want to have additional properties in a flow. They can add it here.